alfresco-search/src/test/java/org/alfresco/solr/query/afts/requestHandler/AFTSDefaultTextQueryTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2005-2019 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.alfresco.solr.query.afts.requestHandler;
+
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.namespace.QName;
+import org.alfresco.solr.client.PropertyValue;
+import org.alfresco.solr.client.StringPropertyValue;
+import org.alfresco.solr.query.afts.TestDataProvider;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.stream.IntStream.range;
+import static junit.framework.TestCase.assertEquals;
+import static org.alfresco.model.ContentModel.*;
+import static org.alfresco.solr.AlfrescoSolrUtils.addNode;
+
+
+/**
+ * @author elia
+ */
+public class AFTSDefaultTextQueryTest extends AbstractRequestHandlerTest
+{
+
+    @BeforeClass
+    public static void beforeClass() throws Exception
+    {
+        TestDataProvider dataProvider = new TestDataProvider(h);
+
+        String[] names = {
+                "test1",
+                "test2",
+                "file3",
+                "name4"};
+
+        String[] descriptions = {
+                "description of test 1",
+                "description 2",
+                "this is not a description of test 1 and 2",
+                "other description"};
+
+        String[] contents = {
+                "test",
+                "content of test 2",
+                "other contents",
+                "content of file number 4"};
+
+        String[] titles = {
+                "Title1",
+                "Other Title",
+                "Third",
+                "Forth"};
+
+        // Creators should not be searched in default queries
+        String[] creators = {
+                "Luca",
+                "Mario",
+                "Giovanni",
+                "Giuseppe"};
```

Comment:
Maybe this information would be easier to read as a list of maps?  I'm not sure tbh.
```
List<Map<String, String>> data = asList(of("name", "test1", "description", "description of test 1", ...),
                                        of("name", "test2", "description", "description 2", ...),...
```

Reply:
I changed as you suggested. It's actually easier to read in this way. Thanks for the hint.